packages/builder/src/internal/core/config/default-config.ts

```diff
@@ -1,22 +1,16 @@
-import { AlgobConfig, HttpNetworkConfig } from "../../../types";
+import type { AlgobChainCfg, AlgobConfig } from "../../../types";
 import { ALGOB_CHAIN_NAME } from "../../constants";
 
-const DEFAULT_NETWORK_CONFIG: HttpNetworkConfig = {
-  // accounts: [],
-  chainName: "devnet",
-  host: "http://localhost",
-  port: 8080,
-  token: "TODO--token-from-devnet"
+const cfg: AlgobChainCfg = {
+  accounts: [],
+  chainName: ALGOB_CHAIN_NAME,
+  throwOnTransactionFailures: true,
+  throwOnCallFailures: true
 };
 
 const defaultConfig: AlgobConfig = {
   networks: {
-    default: DEFAULT_NETWORK_CONFIG,
-    [ALGOB_CHAIN_NAME]: {
-      chainName: ALGOB_CHAIN_NAME,
-      throwOnTransactionFailures: true,
-      throwOnCallFailures: true
-    }
+    [ALGOB_CHAIN_NAME]: cfg
```

Comment:
This looks like it's an array. Can you remove square brackets?
It's not a type.

Reply:
No, it's not an array, and it won't work without the brackes. It's a language feature, special syntax to get a name from a variable.